### ISSUE TITLE
Incompatible version of Poetry.el on Melpa Stable

### ISSUE THREAD

**Issue by crawler-n** [cybniv/poetry.el#55]
Version of Poetry still on Melpa Stable indicates `0.1.0`. This version is incompatible because of uninstallable `cl 1.0` package.

Help from list-packages:

```
Package poetry is incompatible.

     Status: Incompatible because it depends on uninstallable packages.
    Archive: stable
    Version: 0.1.0
     Commit: 6fb34742a66421c6104e02d65e8d391fc4494ab5
    Summary: poetry in Emacs
   Requires: transient-0.1.0, cl-1.0 (not available), xterm-color-1.8
    Website: https://github.com/galaunay/poetry.el
   Keywords: python tools 
 Maintainer: Gaby Launay <gaby.launay@protonmail.com>
     Author: Gaby Launay <gaby.launay@protonmail.com>
Other versions: 20240103.947 (installed), 20240103.947 (melpa).

This package offers an interface to poetry (https://poetry.eustace.io/),
a Python dependency management and packaging command line tool.
```

This is the same as #19 . Seems like `0.2.0` release was not tagged. I don't know anything about Melpa Stable, so not sure if it only pulls from the latest tagged releases.

Would help people who use Melpa Stable if next release (or old one) can be tagged.

**Comment by cybniv** [comment]
closing in favor of https://github.com/cybniv/poetry.el/issues/19, which has been reopened.
